README.md

```diff
@@ -73,12 +73,67 @@ gulp.task('default', gulp.series('clean', 'all'));
 
 ## Incremental Builds
 
-We recommend these plugins:
+You can filter out unchanged files between runs of a task using 
+`gulp.src` `since` option and `gulp.lastRun`:
+```js
+gulp.task('scripts', function() {
+  return gulp.src(paths.scripts, {since: gulp.lastRun('script')})
+    .pipe(sourcemaps.init())
+      .pipe(coffee())
+      .pipe(uglify())
+      .pipe(concat('all.min.js'))
+    .pipe(sourcemaps.write())
+    .pipe(gulp.dest('build/js'));
+});
 
-- [gulp-changed](https://github.com/sindresorhus/gulp-changed) - only pass through changed files
-- [gulp-cached](https://github.com/wearefractal/gulp-cached) - in-memory file cache, not for operation on sets of files
-- [gulp-remember](https://github.com/ahaurw01/gulp-remember) - pairs nicely with gulp-cached
-- [gulp-newer](https://github.com/tschaub/gulp-newer) - pass through newer source files only, supports many:1 source:dest
+gulp.task('images', function() {
+  return gulp.src(paths.images, {since: gulp.lastRun('images')})
+    .pipe(imagemin({optimizationLevel: 5}))
+    .pipe(gulp.dest('build/img'));
+});
+
+gulp.task('watch', function() {
+  gulp.watch(paths.scripts, 'scripts');
+  gulp.watch(paths.images, 'images');
+});
+```
+When run by `gulp.watch`, the `script` and `images` tasks will only rebuild
```

Comment:
When run by ~> When you run 

`When you run `gulp.watch`, the `script` and `images` tasks will only rebuild the files that have been modified since the task was last run.`